aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/s3/S3ExpressDecorator.kt

```diff
@@ -294,6 +294,7 @@ private fun s3ExpressModule(runtimeConfig: RuntimeConfig) =
 
 private fun s3ExpressDependencies(runtimeConfig: RuntimeConfig) =
     arrayOf(
+        CargoDependency.AHash,
```

Comment:
Add that same comment from the Cargo.toml here.

Reply:
Added in f80596c